adr/003-gradle.md

```diff
@@ -18,7 +18,25 @@ Accepted.
 - Gradle is faster with its job execution.
 
 
-### Related Issues
+## Impact
+
+### Positive
```

Comment:
Some of these points end with periods and some don't. I don't particularly have a preference for one style over the other, as long as it's consistent (this comment applies to all the ADRs not just this one)

Reply:
I think because of the bullet-pointed style, I think no periods are best practice. And yes - this was on my mind too.